script/make_utils/check_pytest_determinism.sh

```diff
@@ -40,32 +40,25 @@ set -e
 diff "${OUTPUT_DIRECTORY}/one.txt" "${OUTPUT_DIRECTORY}/two.txt" -I "passed in"
 echo "Successful determinism check"
 
-# Now, check --forcing_random_seed, i.e., check that one can reproduce conditions of a bug in a single file
-# and test without having to relaunch the full pytest, by just picking the right --forcing_random_seed
+# Now, check that one can reproduce conditions of a bug in a single file
+# and test without having to relaunch the full pytest
 
 # All lines that start with "tests/seeding/test_seeding.py::" represent a test that passed
 LIST_FILES=$(grep "tests/seeding/test_seeding.py::" "${OUTPUT_DIRECTORY}/one.txt")
-LIST_SEED=()
```

Comment:
no more need to find --forcing_random_seed in the log files! it was especially boring since sometimes, they were not there if a crash occurred.